pkg/outputs/mysql/mysql.go

```diff
@@ -166,6 +166,38 @@ func (output *MySQLOutput) Execute(msgs []*core.Msg) error {
 				return nil
 			}
 			switch node := msg.DdlMsg.AST.(type) {
+			case *ast.CreateDatabaseStmt:
```

Comment:
The logic for `CreateDatabaseStmt` and `DropDatabaseStmt` seems to be the same, can we refactor it out to use the same function?

Reply:
they don't have common type. how can we set the fields without type assertion?